docs/rest_api.rst

```diff
@@ -640,6 +640,47 @@ Possible Responses
 | 500 Server Error | Internal Raiden node error|
 +------------------+---------------------------+
 
+Querying connections details
+------------------------------
+
+You can query for details of previously joined token networks by making a GET request to the connection endpoint.
+
+``GET /api/<version>/connection``
+
+The request will return a JSON object where each key is a token address for which you have open channels.
+The values are a JSON object containing numeric values for `funds` from last connect request, `sum_deposits`
```

Comment:
here and in the rest of the paragraph please use double backticks:

So 

```
``funds``
```

and so on